### PR TITLE
Retag jupyter images with tag v0.5.0

### PR DESCRIPTION
**Pull request by zabbasi**
built tf 2.0 and 1.13 images
retag all images to v0.5.0

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2805)

### CODE REVIEW

**Comment by zabbasi** [comment]
//assign @lluunn

File: kubeflow/jupyter/config.yaml [hunk 1]
```diff
@@ -21,25 +21,29 @@ spawnerFormDefaults:
   image:
     # The container Image for the user's Jupyter Notebook
     # If readonly, this value must be a member of the list below
-    value: gcr.io/kubeflow-images-public/tensorflow-1.10.1-notebook-cpu:v-base-74ac37c-1214
+    value: gcr.io/kubeflow-images-public/tensorflow-1.10.1-notebook-cpu:v0.5.0
```
Comment (lluunn) [inline kubeflow/jupyter/config.yaml]: This is the default?
If so, it's better to use a newer, 1.13.1 or 1.12.0

**Pull request merged**